DOC-2063: Added TINY-9633 to the release notes. (#2776)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

{productname} 6.5 also includes the following improvements:

=== Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
//#TINY-9633

> In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab on the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.

Reviewer: ```suggestion
In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
```

> In {productname} 6.5, a new pipeline has been implemented for delivering translations for both long strings and professional translations for text within the “Keyboard Navigation” tab inside the Help dialog.

Reviewer: ```suggestion
In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the “Keyboard Navigation” tab inside the Help dialog have also been procured.

> As a result, the text is now shown in the language specified by the language option.

Reviewer: ```suggestion
As a result, the text in the Help dialog's "Keyboard Navigation" tab is now shown in the language specified by the `language` option.
```

> === Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
> //#TINY-9633
> 
> In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered. Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the “Keyboard Navigation” tab.
> 
> In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the “Keyboard Navigation” tab inside the Help dialog have also been procured.
> 
> As a result, the text in the Help dialog's "Keyboard Navigation" tab is now shown in the language specified by the `language` option.
> 
> [NOTE]
> Due to the large differences between this new pipeline and conventional methods of delivering translations, the professional translations for the text on the “Keyboard Navigation” tab have been made **open-source** in the `i18n` subdirectory in the Help plugin for _community users_.

Reviewer 2: ```suggestion
=== Translations added for Help text displayed at **Help > Help > Keyboard Navigation**.
//#TINY-9633

In previous versions of {productname}, translations for the text on the “Keyboard Navigation” tab in the Help dialog were not available due to a flaw with the way translations were delivered.

Conventionally, translations were parsed into JavaScript files containing objects which were keyed by the English string, with the value being the translated equivalent in a different language. This was unsuitable for very long strings such as the text on the *Keyboard Navigation* tab.

In {productname} 6.5, a new pipeline has been implemented for delivering translations of long strings. Professional translations of text within the *Keyboard Navigation* tab inside the **Help** dialog have also been procured.

As a result, the text in the **Help** dialog’s *Keyboard Navigation* tab is now shown in the language specified by the current editor’s `language` option.

NOTE: Due to the large differences between this new pipeline and conventional methods of delivering translations, the professional translations for the text on the *Keyboard Navigation* tab have been made open source for community users. These translations are in the `i18n` subdirectory of the **Help** plugin.
```

Copy-edits.
